src/winetricks

```diff
@@ -11971,6 +11971,59 @@ __EOF__
 
 #----------------------------------------------------------------
 
+w_metadata office2013pro apps \
+    title="Microsoft Office 2013 Professional" \
+    publisher="Microsoft" \
+    year="2013" \
+    media="cd" \
+    file1="setup.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Microsoft Office/Office15/WINWORD.EXE"
+
+load_office2013pro()
+{
+    w_package_unsupported_win64
+
+    if [ ! -x "$(which ntlm_auth 2>/dev/null)" ]; then
+        w_die "winbind (part of samba) is required for the installation"
+    fi
+
+    #link from https://www.askvg.com/direct-download-link-microsoft-office-2013-professional-plus-free-trial/
+    w_download http://care.dlservice.microsoft.com/dl/download/2/9/C/29CC45EF-4CDA-4710-9FB3-1489786570A1/OfficeProfessionalPlus_x86_en-us.img 236f8faae3f979ec72592a63784bba2f0d614916350c44631221b88ae9dae206 "OFFICE15.iso"
+
+    w_set_winver win7
+
+    w_call corefonts
+    w_call riched20
+
+
+    if w_workaround_wine_bug 43581 "wine's libxml2 has problems parsing some regex strings"; then
+        w_call msxml6
```

Comment:
Should say wine's msxml6, libxml2 is a separate project

Reply:
wine is using libxml2 and that was causing the [bug](https://bugs.winehq.org/show_bug.cgi?id=43581). 

https://github.com/Winetricks/winetricks/pull/939/commits/58e5591c2d21b35d853a4982ee650d031523762b

Reply:
Right. The bug is real, I don't doubt that.  I'm saying libxml2 isn't part of wine. The wine components are msxml{3,4,6}

Reply:
https://github.com/Winetricks/winetricks/pull/939/commits/94f402af6f9038abe16d5f2457a041c40c53f27b#diff-90b1fb64c3f0a6c10d15531fb2008296R11999

not sure how comment it correctly. Should be fine like this now.